### PR TITLE
SW-4839 Add the ability to unselect the project in the accession create form

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #2149** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2149?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2149?utm_source=stack-comment).

File: src/scenes/AccessionsRouter/Accession2CreateView.tsx [hunk 1]
```diff
@@ -227,6 +229,7 @@ export default function CreateAccession(): JSX.Element {
                 record={record}
                 setRecord={setRecord}
                 availableProjects={availableProjects}
+                allowUnselect
```
Comment (karthikbtf) [inline src/scenes/AccessionsRouter/Accession2CreateView.tsx:232]: Do we need a similar option for accession edit?

**Review by karthikbtf — APPROVED** [review]
Nice!

File: src/scenes/AccessionsRouter/Accession2CreateView.tsx (hunk 1 above)
Comment (nickgraz) [reply src/scenes/AccessionsRouter/Accession2CreateView.tsx:232]: I'm not really sure, it seems like it should be though. There isn't any mention of that in the designs, and the PRD technically only [mentions](https://terraformation.atlassian.net/wiki/spaces/PM/pages/811303013/PRD+Projects#As-a-manager-or-an-admin-of-an-organization-I-want-to-add%2Fchange-a-project-of-an-existing-accession%2Fseedling-batch%2Fplanting-site) "add or change a project".

Comment (karthikbtf) [reply src/scenes/AccessionsRouter/Accession2CreateView.tsx:232]: Got it, maybe a question for @chudo76

Comment (nickgraz) [reply src/scenes/AccessionsRouter/Accession2CreateView.tsx:232]: Doing this in a follow up PR now

**Pull request merged**